__tests__/Auth0Client/getTokenSilently.test.ts

```diff
@@ -663,6 +664,68 @@ describe('Auth0Client', () => {
       );
     });
 
+    describe('concurrency', () => {
+      it('should call _getTokenSilently multiple times when no call in flight concurrently', async () => {
```

Comment:
In order to test the promise concurrency fix, we can not rely on the actual HTTP call as this will be avoided by the locking library. So instead of testing that, we are verifying that `_getTokenSilently ` is or isn't called as expected.